src/site/topic24.rst

```diff
@@ -44,6 +44,60 @@ Constructors
 Static Node Class
 =================
 
+* You likely noticed the use of ``Node<T> root`` 
+* Like the linked stack and linked queue, we will make use of nodes
+* You may remember that we had two options for doing this
+    1. Simply create a top level node class
+    2. Create a :doc:`nested node class aside. </topic8-nested>`
+
+* To keep things clean, we will go with option #2
+
+.. code-block:: java
+    :linenos:
+
+    // This class exists within the LinkedBinarySearchTree class
+    private static class Node<T> {
```

Comment:
Writing getters and setters is ever so slightly triggering. Really only enough to write this comment though.